tests/test_factstore.py

```diff
@@ -1,7 +1,62 @@
-from fact_store.app import FactStore
+from fact_store.fact_store_api import FactStore
```

Comment:
I am not sure how useful these tests are - you are basically just trying out the SQLAlchemy and making sure it creates the tables as defined and can perform an insert and select - i.e. testing SQLAlchemy itself rather than your own code

Reply:
The test cases are there to validate that when I want to store feedback that it persists. Also checking to make sure that we don't get duplicate feedback. Please give the code another read.

Reply:
I've reduced the amount of repeat tests and will keep it simple for now. Is this a blocker?